CLAUDE.md

```
@@ -490,23 +490,26 @@ If tests fail or clippy reports issues, the implementation is NOT complete until
 1. When given an URL - parse the incoming URL
 
 ```text
-https://github.com/ophidiarium/cribo/pull/<PR#>#discussion_r<COMMENT_ID>
+https://github.com/ophidiarium/cribo/pull/<PR_NUMBER>#discussion_r<COMMENT_ID>
 ```
 
-- `<PR#>` → digits between `/pull/` and `#`.
+- `<PR_NUMBER>` → digits between `/pull/` and `#`.
 - `<COMMENT_ID>` → digits after `discussion_r`.
 
 2. Fetch the comment body
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/2/3

(MD029, ol-prefix)